common/src/main/java/com/wynntils/functions/generic/RandomFunctions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions.generic;
+
+import com.wynntils.core.functions.GenericFunction;
+import com.wynntils.core.functions.arguments.FunctionArguments;
+import java.util.List;
+
+public class RandomFunctions {
+    public static class RandomFunction extends GenericFunction<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            int min = arguments.getArgument("min").getIntegerValue();
+            int max = arguments.getArgument("max").getIntegerValue();
+            return (int) ((Math.random() * (max - min)) + min);
+        }
+
+        @Override
+        public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
+            return new FunctionArguments.RequiredArgumentBuilder(List.of(
+                    new FunctionArguments.Argument<>("min", Integer.class, null),
+                    new FunctionArguments.Argument<>("max", Integer.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("rand", "rnd");
+        }
+    }
+
+    public static class RandomMaxFunction extends GenericFunction<Integer> {
```

Comment:
Is this a copy paste of random with max being inclusive? (While random is min exclusive and max inclusive, as it should be)

Reply:
I don't really see any value in keeping both. Just have the default random implementation, like every language/etc does.

Reply:
I agree. And it might just as well live in Math functions, no need to create a new top-level class for that.

Reply:
I did mainly random with range because there is probably no use of just a random number between 0 and 1, and it would just make functions much, much longer when it would be required to write min/max implementation manually every time.
How about, then, one function that returns a double value but with max exclusive, and a second one that return an integer value with max inclusive? The thing is that I don't know what use cases it would have, but some people on discord requested it, meaning they have some ideas and I don't know what version would suite them the most, so I think there is no harm in keeping both.

Reply:
Can you explain the use case for a random function in the first place? I am not sure I understand why it is even needed.

Reply:
I just saw 3 people in total request it, so I decided to implement it. Some time ago someone told me they would use it for flashing text, for example in like changing colors.